### PR TITLE
refactor: use getElementById whenever possible

### PR DESCRIPTION
**Pull request by saschanaz**
Maybe someone batch-converted jQuery into querySelector?

### CODE REVIEW

File: tests/spec/core/markdown-spec.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ describe("Core - Markdown", function() {
     ops.config.format = "markdown";
     const doc = await makeRSDoc(ops);
     expect(doc.querySelector("code")).toBeFalsy();
-    expect(doc.querySelector("#foo").textContent === "Foo");
+    expect(doc.getElementById("foo").textContent).toBe("1. Foo");
```
Comment (saschanaz) [inline tests/spec/core/markdown-spec.js]: @marcoscaceres, is this expected?

Comment (marcoscaceres) [reply tests/spec/core/markdown-spec.js]: It looks correct... which is odd, considering the old test.

Comment (saschanaz) [reply tests/spec/core/markdown-spec.js]: I think the old test simply didn't work.

Comment (marcoscaceres) [reply tests/spec/core/markdown-spec.js]: Ah, right. It just “expect()”s nothing.

**Pull request merged**